src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/EnumerableViewOfDispatch.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    [Serializable]
+    internal class EnumerableViewOfDispatch : ICustomAdapter, IEnumerable
+    {
+        private const int DispId_NewEnum = -4;
+        private readonly object dispatch;
```

Comment:
If it doesn't need to be serializable, it'd be nice to conform to the naming conventions we have, e.g. this should be _dispatch.